OrbitGl/App.cpp

```diff
@@ -315,16 +314,19 @@ void OrbitApp::AppendSystrace(const std::string& a_FileName,
 
 //-----------------------------------------------------------------------------
 bool OrbitApp::Init() {
-  GOrbitApp = new OrbitApp();
-  GCoreApp = GOrbitApp;
+  GOrbitApp = std::make_unique<OrbitApp>();
+  GCoreApp = GOrbitApp.get();
   GTimerManager = std::make_unique<TimerManager>();
-  GTcpServer = new TcpServer();
+  GTcpServer = std::make_unique<TcpServer>();
```

Comment:
you probably wanted to use make_shared here.

Reply:
solved, see last comment